applications/nrf_desktop/configuration/nrf52833_pca10111/buttons_def.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include "gpio_pins.h"
+
+/* This configuration file is included only once from button module and holds
+ * information about pins forming keyboard matric.
```

Comment:
`keyboard matrix`, this applies to all the boards

Reply:
Added a commit with a fix for this typo. Thanks